<launch>
  <node name="smc_server_front" pkg="naro_smc_srvs" type="smc_server" respawn="true">
    <rosparam command="load" file="$(find naro_smc_srvs)/config/smc_server_front.yaml"/>
    <param name="configuration/file" value="$(find naro_smc_srvs)/config/smc_front.xml"/>
  </node>
  <node name="smc_server_rear" pkg="naro_smc_srvs" type="smc_server" respawn="true">
    <rosparam command="load" file="$(find naro_smc_srvs)/config/smc_server_rear.yaml"/>
    <param name="configuration/file" value="$(find naro_smc_srvs)/config/smc_rear.xml"/>
  </node>

    <node name="hallSensor_front" pkg="naro_sensor_srvs" type="hall_sensor" output="screen">
    <rosparam command="load" file="$(find naro_sensor_srvs)/config/hallSensor_front.yaml"/>
  </node>
  <node name="hallSensor_rear" pkg="naro_sensor_srvs" type="hall_sensor" output="screen">
    <rosparam command="load" file="$(find naro_sensor_srvs)/config/hallSensor_rear.yaml"/>
  </node>
  
  <node name="tankCtrl" pkg="naro_tank_ctrl" type="tank_ctrl" output="screen">
  </node>

</launch>